ConfigurationSystem/Client/Utilities.py

```diff
@@ -486,11 +486,9 @@ def getDBParameters( fullname, defaultHost = 'localhost',
   'db' and 'queueSize'
   """
 
-  fullname = fullname
```

Comment:
The logic with default values in this method is not good. It is often the case that the DB CS parameters are not defined for whatever reason ( human error, CS unavailable, etc ). This is a real bug which will prevent the DB from working. This is the only case where we allow exceptions ( at object initialization ). So, the defaults must be dropped here. I will make changes ( I have somthing else to commit as well ) and we will see